packages/core/src/payment/strategies/checkoutcom-custom/checkoutcom-custom-payment-strategy.ts

```diff
@@ -46,7 +46,11 @@ export default class CheckoutcomCustomPaymentStrategy extends CreditCardPaymentS
             await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
             await form.submit(payment);
         } catch (error) {
-            return this._processResponse(error);
+            if (error instanceof RequestError) {
+                return this._processResponse(error);
```

Comment:
All errors should be passed to `this._processResponse`, so lets update error interface with `unknown`